src/Docker/DatabaseServices.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Docker;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class DatabaseServices
+{
+    public static function getDatabaseSkeleton(string $name, string $version): array
+    {
+        switch (strtolower($name)) {
+            case 'mariadb':
+                return [
+                    'image' => sprintf('mariadb:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306'],
+                ];
+            case 'mysql':
+                return [
+                    'image' => sprintf('mysql:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306'],
+                ];
+            case 'postgres':
+                return [
+                    'image' => sprintf('postgres:%s', $version),
+                    'environment' => [
+                        'POSTGRES_PASSWORD' => 'main',
+                        'POSTGRES_USER' => 'main',
+                        'POSTGRES_DB' => 'main',
+                    ],
+                    'ports' => ['5432'],
+                ];
+        }
+    }
+
+    public static function getDefaultPorts(string $service): array
+    {
+        $ports = [];
+
+        switch (strtolower($service)) {
+            case 'mariadb':
+            case 'mysql':
+                $ports = ['3306'];
+                break;
+            case 'postgres':
+                $ports = ['5432'];
+                break;
+        }
+
+        return $ports;
+    }
+
+    public static function getSuggestedServiceVersion(string $name): string
+    {
+        $name = strtolower($name);
```

Comment:
Same here - are we guarding against some user input that might us capitals? Or is that case not really possible?

Reply:
exactly what we're doing here but i just realized `case` is doing ` ==` not `===` so I can get rid of those here and above..

Reply:
done

Reply:
to clarify, this is done because we present via `$io` as `MariaDB` or `MySQL` in `interact()`, user selects `0`, `1`, `2` and we ultimately pass the string representation to `DatabaseServices::getDatabaseSkeleton()` or any of the other methods..

Reply:
Ah, ok - that's the explanation I was missing - *why* we needed this in the first place. I'd vote for normalizing it immediately after we ask the question. Then we can keep things "tight" here.

Reply:
done